frontend/src/components/Answer/Answer.tsx

```diff
@@ -47,12 +63,7 @@ export const Answer = ({
                             title="Show thought process"
                             ariaLabel="Show thought process"
                             onClick={() => onThoughtProcessClicked()}
-                            disabled={
-                                !chatResponse.classification &&
-                                !chatResponse.answer.query &&
-                                !chatResponse.answer.query_generation_prompt &&
-                                !chatResponse.answer.query_result
-                            }
+                            disabled={!chatResponse.classification && !answer.query && !answer.query_generation_prompt && !answer.query_result}
```

Comment:
I know this was just cleanup, but should this be a function? Fine to stay as is, but a conditional this long we should move.

Reply:
fair. I like that idea.